src/uct/ib/ud/base/ud_def.h

```diff
@@ -32,7 +32,7 @@
  * so add 1 to the max/min window constants instead of doing this in the code
  */
 #define UCT_UD_CA_MIN_WINDOW    2
-#define UCT_UD_CA_MAX_WINDOW    1025
+#define UCT_UD_CA_MAX_WINDOW    513
```

Comment:
it can have negative effect on point-to-point bandwidth, @alex-mikheev pls comment